special-pages/pages/new-tab/app/favorites/components/TileRow.js

```diff
@@ -44,11 +44,11 @@ export const TileRow = memo(
                     Array.from({ length: fillers }).map((_, fillerIndex) => {
                         // first is always the + (plus) button
                         if (fillerIndex === 0) {
-                            return <PlusIconMemo key="placeholder-plus" onClick={add} />;
+                            return <PlusIconMemo key={`placeholder-plus-${items.length}`} onClick={add} />;
```

Comment:
this treatment is needed to ensure webkit will unmount the components correctly. This helps to ensure that whenever the placeholder icon is moved, the element is invalidated and cleans up the DOM.